paasta_tools/kubernetes_tools.py

```diff
@@ -1445,3 +1475,13 @@ def sanitise_service_name(
     service: str,
 ) -> str:
     return service.replace('_', '--')
+
+
+def load_custom_resources(
```

Comment:
this is actually `load_custom_resource_definitions` isn't it?

Reply:
done